core/iwasm/fast-jit/jit_ir.h

```diff
@@ -409,6 +418,10 @@ _jit_insn_new_LookupSwitch_1(JitOpcode opc, JitReg value, uint32 num);
 #undef ARG_LIST_VReg_2
 #undef ARG_DECL_LookupSwitch_1
 #undef ARG_LIST_LookupSwitch_1
+#if WASM_ENABLE_SHARED_MEMORY != 0
+#undef ARG_DECL_Null_0
+#undef ARG_LIST_Null_0
```

Comment:
Prefer to define ARG_DECL_Reg_0 and ARG_LIST_Reg_0, see the patch below:
[jit_ir.zip](https://github.com/bytecodealliance/wasm-micro-runtime/files/11209061/jit_ir.zip)